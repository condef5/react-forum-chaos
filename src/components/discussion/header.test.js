import React from "react";
import { render } from "@testing-library/react";

import Header from "./header";

test("Header component", () => {
  const { asFragment } = render(<Header discussion={[]} />);
  expect(asFragment()).toMatchSnapshot();
});
