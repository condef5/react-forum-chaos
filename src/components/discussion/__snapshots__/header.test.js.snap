// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component 1`] = `
<DocumentFragment>
  .emotion-3 {
  margin: 5px;
  border-bottom: 2px dashed #34495e;
}

.emotion-0 {
  font-size: 3em;
  margin: 10px 0;
}

.emotion-1 {
  font-size: 1em;
  color: #34495e;
}

.emotion-2 {
  font-size: 1.3em;
  color: #34495e;
}

<div
    class="emotion-3"
  >
    <h1
      class="emotion-0"
    />
    <span
      class="emotion-1"
    >
       - 
    </span>
    <p
      class="emotion-2"
    />
  </div>
</DocumentFragment>
`;
